app/assets/javascripts/discourse/app/components/sidebar/more-section-links.js

```diff
@@ -72,8 +72,8 @@ export default class SidebarMoreSectionLinks extends GlimmerComponent {
   }
 
   @action
-  toggleSectionLinks() {
-    this.shouldDisplaySectionLinks = !this.shouldDisplaySectionLinks;
+  toggleSectionLinks(element) {
+    this.shouldDisplaySectionLinks = element.target.hasAttribute("open");
```

Comment:
```suggestion
  toggleSectionLinks(event) {
    this.shouldDisplaySectionLinks = event.target.hasAttribute("open");
```

I think this should be `event`?